common/culture/cultures/demonic.txt

```diff
@@ -336,6 +336,8 @@
 	nathrezim = {
```

Comment:
New markings have to be localized.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/131697905-44ab18c4-2b76-4d4c-a5ac-5f35fc5515c6.png)

</details>

Reply:
Done in 7c279b049bdbbb55a659c447dd9475a5bbe48b63.